### PR TITLE
Rename keyboardIcon => openPickerIcon

### PR DESCRIPTION
**Pull request by Philipp91**
See #1693 [step 1](https://github.com/mui-org/material-ui-pickers/issues/1693#issuecomment-616758267).

### CODE REVIEW

**Comment by Philipp91** [comment]
Is there an ongoing list of breaking changes where I should be adding this?

**Review by dmtrKovalenko — APPROVED** [review]

File: lib/src/_shared/icons/CalendarIcon.tsx [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
 import { createSvgIcon } from '@material-ui/core/utils';
 
-export const KeyboardIcon = createSvgIcon(
+export const CalendarIcon = createSvgIcon(
   <>
     <path d="M17 12h-5v5h5v-5zM16 1v2H8V1H6v2H5c-1.11 0-1.99.9-1.99 2L3 19c0 1.1.89 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2h-1V1h-2zm3 18H5V8h14v11z" />
     <path fill="none" d="M0 0h24v24H0z" />
   </>,
-  'KeyboardIcon'
+  'CalendarIcon'
```
Comment (oliviertassinari) [inline lib/src/_shared/icons/CalendarIcon.tsx]:
```suggestion
  'Calendar'
```

Comment (oliviertassinari) [reply lib/src/_shared/icons/CalendarIcon.tsx]: https://github.com/mui-org/material-ui/blob/0298ee013da2d8e7e8944cb18471da08e02f87b3/packages/material-ui/src/utils/createSvgIcon.js#L17

**Comment by Philipp91** [comment]
Do you generally prefer additional commits pushed behind the existing ones, or commit amendments?

**Comment by dmtrKovalenko** [comment]
Coverage mismatch is actually caused by missing cypress coverage #1507. So we can ignore it

**Pull request merged**